ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -39,7 +43,21 @@ type ContainerDef struct {
 	Essential bool `yaml:"essential"`
 }
 
-func readECSParams(filename string) (*ECSParams, error) {
+// Non-TaskDefinition specific parameters
+type RunParams struct {
+	NetworkConfiguration NetworkConfiguration `yaml:"network_configuration"`
+}
+
+type NetworkConfiguration struct {
+	AwsVpcConfiguration AwsVpcConfiguration `yaml:"awsvpc_configuration"`
+}
+
+type AwsVpcConfiguration struct {
```

Comment:
nit: go initialism will tell you that this should be `AWSVPCConfiguration`

Reply:
such capital letters. wow.